app/src/pages/inside/launchesPage/LaunchToolbar/actionPanel/actionPanel.jsx

```diff
@@ -80,26 +86,37 @@ export class ActionPanel extends Component {
     {
       label: this.props.intl.formatMessage(messages.actionMerge),
       value: 'action-merge',
+      visible: !this.props.debugMode,
```

Comment:
I think it's better to make items visible by default, and use property 'hidden' if we want to hide an item